gradle-plugin/build.gradle

```diff
@@ -44,7 +44,7 @@ gradlePlugin {
 publishing {
     publications.withType(MavenPublication) {
         pom {
-            name = "pekko-grpc-gradle-plugin"
+            name = "Apache Pekko Grpc Gradle Plugin"
```

Comment:
There is a maven plugin too

Reply:
> There is a maven plugin too

Yes but the maven plugin is generated by sbt so the name will also be fixed by the updated sbt-apache-sonatype